src/client.h

```diff
@@ -400,6 +399,7 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 
 	ClientScripting *getScript() { return m_script; }
 	const bool moddingEnabled() const { return m_modding_enabled; }
+	const bool moddingLoaded() const { return m_mods_loaded; }
```

Comment:
`modsLoaded()` would be a better fitting name IMO (since we're already touching the code)

Reply:
Ok.

Reply:
Done.